discovery-provider/src/api/v1/playlists.py

```diff
@@ -34,14 +55,33 @@ class Playlist(Resource):
     def get(self, playlist_id):
         """Fetch a playlist."""
         playlist_id = decode_with_abort(playlist_id, ns)
-        args = {"playlist_id": [playlist_id], "with_users": True}
-        playlists = get_playlists(args)
-        playlists = list(map(extend_playlist, playlists))
+        playlists =  get_playlist(playlist_id, None)
```

Comment:
should this be playlist = get_playlist?

Reply:
yeah this is weird, get_playlist returns a single playlist in an array. I'll fix this